### DISCUSSION TITLE
关于播放音频时播放列表的问题

### DISCUSSION THREAD

**Discussion by xhhdd** [AlistGo/alist#529] [Q&A]
大佬你好！

点击音频文件播放的时候，我发现alist会根据当前文件夹的文件自动创建一个播放列表。
如下图所示：
<img width="547" alt="截屏2022-02-12 下午2 51 44" src="https://user-images.githubusercontent.com/91271854/153700674-ccc88979-55af-4b61-b7c3-ce885c753107.png">

这是非常正常的，按默认的数字进行一个排列。
但是我如果点击第9条音频，播放列表中第9条在最前面，然后接着又回到第1条，第2条...
如下图所示：

<img width="527" alt="截屏2022-02-12 下午2 55 35" src="https://user-images.githubusercontent.com/91271854/153700783-5e29375e-4327-452c-97ad-3f385d7e03a8.png">

**这是我的第一个问题：请问能不能点击第9条音频之后，接下来就是第10条，第11条按顺序。**

好叭，上面的第一个问题我觉得可能有点私人问题的意思。

**接下来是我的第二个问题：在移动端的播放列表无法上下拖动。**
我是ios端，测试了一下safari，chrome,qq浏览器等，都无法上下拖动播放列表。
![IMG_931524732709-1](https://user-images.githubusercontent.com/91271854/153701000-b3ace40a-cee4-4418-9517-e514b630f291.jpeg)
如上图所示，播放列表中有100条音频，但是我只能看到界面里显示的这些，我没法往下滑动播放列表，只能改变现有界面里音频的顺序。
我在电脑上的浏览器上打开，这个播放列表都是没有问题的，可以正常操作。
以下是我搭建的alist，可以复现这个问题。
[复现链接](https://file.xhhdd.cc/Aurality/ear_training/chord/1642258435)

打扰了 谢谢大佬
新年快乐！~

**Comment by DeepFal** [comment]
拖动右侧的上下进度条或许可以暂时解决你的问题(有点窄容易按不到)

**Reply by xhhdd** [reply]
尴尬~我手机上右侧好像没有进度条

**Reply by DeepFal** [reply]
打开播放列表后右侧会有 
![](https://user-images.githubusercontent.com/41166795/153704723-455850e0-5fda-4a81-8226-05e187439be6.jpg)
#

**Reply by xhhdd** [reply]
恩恩 可能系统不同😂所以没有 我的图发在上面 试了好几个浏览器都没有右边的条

**Comment by xhofe** [comment]
https://github.com/alist-org/alist-web/commit/8e761e6da5a33b787f8d12ed3314b9d97c1b48fc
https://github.com/alist-org/alist-web/commit/26f079137d3d4251ad9c91b5057dad944e359f61

**Reply by xhhdd** [reply]
> [alist-org/alist-web@8e761e6](https://github.com/alist-org/alist-web/commit/8e761e6da5a33b787f8d12ed3314b9d97c1b48fc) [alist-org/alist-web@26f0791](https://github.com/alist-org/alist-web/commit/26f079137d3d4251ad9c91b5057dad944e359f61)

！谢谢你  
我看到那个标题好像就是我说的这个两个问题
不过 诶 我现在可以用到这个版本吗
我是用docker安装alist的

不好意思哈 我是小白😂

**Reply by xhofe** [reply]
暂时还不行，release之后可用。

**Reply by xhhdd** [reply]
> 暂时还不行，release之后可用。

好嘞好嘞 谢谢你！

**Comment by im-dashan** [comment]
你的这个音乐播放器的样式怎么弄的，

**Reply by xhhdd** [reply]
很早之前的alist版本了
